README.md

```diff
@@ -14,7 +14,7 @@ Enable swipe action on a list view
  - List view click action and button on the foreground can be used as normal
 
 ## Dependencies
-Mendix 6.10 or 7.
+Mendix 6.10 or 7.18

```

Comment:
NF support is from >= 7.14
6 will never support NF

Reply:
Done